### ISSUE TITLE
Mark up lists of links as unordered lists

### ISSUE THREAD

**Issue by eatyourgreens** [zooniverse/front-end-monorepo#667]
**Package**

**Feature or Issue Description**

**To Dos**

- [ ] Example

**Comment by eatyourgreens** [comment]
[`aria-current`](https://tink.uk/using-the-aria-current-attribute/) looks useful too.

**Comment by rogerhutchings** [comment]
Zooniverse site navigation:

<img width="508" alt="Screenshot 2019-09-14 11 44 27" src="https://user-images.githubusercontent.com/2725841/64907013-467d9180-d6e5-11e9-9d1a-0914b8383307.png">

**Comment by rogerhutchings** [comment]
User profile dropdown menu:

<img width="988" alt="Screenshot 2019-09-14 11 48 41" src="https://user-images.githubusercontent.com/2725841/64907051-b4c25400-d6e5-11e9-82e6-ff59caa02750.png">

**Comment by eatyourgreens** [comment]
Dropdown menus should be navigable with up/down keys too. The user menu does do this in PFE.

**Comment by eatyourgreens** [comment]
Lists of links on the publications page aren’t marked up as lists either.
